tests/integration/tables/CMakeLists.txt

```diff
@@ -351,11 +351,21 @@ function(generateTestsIntegrationTablesTestsTest)
     thirdparty_googletest
   )
 
-  add_test(NAME tests_integration_tables-test COMMAND tests_integration_tables-test)
+  add_test(NAME tests_integration_tables-test COMMAND tests_integration_tables-test --gtest_filter=-memoryDevices.test_sanity)
   set_tests_properties(
     tests_integration_tables-test
     PROPERTIES ENVIRONMENT "TEST_CONF_FILES_DIR=${TEST_CONFIGS_DIR}"
   )
+
+  if(OSQUERY_BUILD_ROOT_TESTS)
+    add_test(NAME tests_integration_tables_root-test COMMAND tests_integration_tables-test --gtest_filter=memoryDevices.test_sanity)
+    set_tests_properties(tests_integration_tables_root-test PROPERTIES LABELS "root-required")
+    set_tests_properties(
+        tests_integration_tables_root-test
+        PROPERTIES ENVIRONMENT "TEST_CONF_FILES_DIR=${TEST_CONFIGS_DIR}"
+    )
+  endif()
+
```

Comment:
This separation it's only needed and used on Linux.
On Windows and macOS there's no separation of the tests and doing like this it means that the `memoryDevices.test_sanity` won't be run on those platforms.